mapper/mapper.go

```diff
@@ -0,0 +1,57 @@
+package mapper
+
+import (
+	"context"
+
+	dynatracev1alpha1 "github.com/Dynatrace/dynatrace-operator/api/v1alpha1"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	InstanceLabel               = "dynakube.dynatrace.com/instance"
+	UpdatedByDynakubeAnnotation = "dynatrace.com/updated-via-operator"
+)
+
+type ConflictCounter struct {
+	i int
+}
+
+func (c *ConflictCounter) Inc(dk *dynatracev1alpha1.DynaKube) error {
+	if !dk.Spec.CodeModules.Enabled {
+		return nil
+	}
+	c.i += 1
+	if c.i > 1 {
```

Comment:
could be changed to just use `bool`, because the count only checks if there's a conflict not how many conflicts there are